data-formats/src/ownershipvoucher.rs

```diff
@@ -441,6 +441,29 @@ impl OwnershipVoucherHeader {
     }
 }
 
+fn check_device_info(device_info: &str) -> Result<bool> {
+    let mut chars = device_info.chars();
+    let result = chars.all(|f| {
+        let acceptable_ascii_range = 32..127;
```

Comment:
Let's make this a global const.

Reply:
Done.

Question: do you want me to move that global const and [the other one](https://github.com/fedora-iot/fido-device-onboard-rs/blob/main/data-formats/src/ownershipvoucher.rs#L58) to the top (after the imports) or do you prefer it to be close to where it's used?

Reply:
Let's move them to the top of the file, just after the imports.

Reply:
Done.